ghost/core/test/e2e-webhooks/pages.test.js

```diff
@@ -0,0 +1,122 @@
+const {
+    agentProvider,
+    mockManager,
+    fixtureManager,
+    matchers
+} = require('../utils/e2e-framework');
+const {
+    stringMatching,
+    anyUuid,
+    anyLocalURL,
+    anyISODateTime,
+    anyObjectId,
+    anyContentVersion,
+    anyNumber
+} = matchers;
+
+const anyGhostAgent = stringMatching(
```

Comment:
The `anyGhostAgent` matcher is not a part of "matcher", so there's no need to re-declare it here 🙃  If you rebase your changes over the latest `main` the matcher should be available 👍

Reply:
Thanks!